app/views/admin/index.html.erb

```diff
@@ -0,0 +1,31 @@
+<h1>Requests</h1>
+
+<div class="admin-requests">
+  <% @requests.each do |origin, collection| %>
+    <% uniq_locations = collection.map { |h| h['origin_location'] }.uniq %>
+
+    <h3><%= SULRequests::Application.config.libraries[origin] %></h3>
```

Comment:
There is now a method on the `LibraryLocation` class we can use instead of the config directly (this wasn't in master when you submitted this PR initially).

https://github.com/sul-dlss/sul-requests/blob/master/app/models/library_location.rb#L22-L24

```ruby
LibraryLocation.library_name_by_code(origin)
```